src/borg/archive.py

```diff
@@ -2338,7 +2343,9 @@ def orphan_chunks_check(self):
                 )
                 for id_ in unused:
                     self.repository.delete(id_)
+                    self.repair_count += 1
```

Comment:
Guess it is *least* important to count the fixed orphaned objects - while they should not occur in the ideal case, there can be conditions producing them.

But, orphans (chunks that are not referenced by any archive) are only a cosmetic issue and we could also just silently remove them (that's why the log message is on INFO level, not WARNING or ERROR).

It would be more important to count the severe "repaired" issues, e.g. if a referenced chunk is missing from the repo and was replaced by an all-zero replacement chunk. That's data loss and the repair is only bringing back consistency, but it's not bringing back the data.

If such a lost chunk reappears later, borg check can also **heal** it, that's a real repair then.